redis_enterprise/manifest.json

```
@@ -57,8 +57,10 @@
       "redis-enterprise-overview": "assets/dashboards/redis_enterprise_overview.json",
       "redis-enterprise-database": "assets/dashboards/redis_enterprise_database.json",
       "redis-enterprise-node": "assets/dashboards/redis_enterprise_node.json",
-      "redis-enterprise-replication": "assets/dashboards/redis_enterprise_replication.json",
-      "redis-enterprise-proxy": "assets/dashboards/redis_enterprise_proxy.json"
+      "redis-enterprise-active-active": "assets/dashboards/redis_enterprise_active-active.json",
```

Changing the file name and Title is fine, but please keep the same `key` here
```suggestion
      "redis-enterprise-replication": "assets/dashboards/redis_enterprise_active-active.json",
```